site/gatsby-site/src/api/parseNews.js

```diff
@@ -0,0 +1,21 @@
+import Mercury from '@postlight/mercury-parser';
+import { format, parseISO } from 'date-fns';
+
+export default async function handler(req, res) {
+  const { url } = req.query;
+
+  const article = await Mercury.parse(url, { contentType: 'text' });
+
+  const response = {
+    title: article.title,
+    authors: [article.author],
```

Comment:
What if there are multiple authors? I think the form is still input as a CSV here?

Reply:
Good question. Do you have an example?

Reply:
Two that may get treated differently

https://www.wsj.com/articles/ukraine-war-leads-to-more-global-business-disruption-11646173915

https://www.theguardian.com/world/2022/mar/01/russias-war-in-ukraine-complete-guide-in-maps-video-and-pictures